themes/nav-community-v1/layouts/partials/footer.html

```diff
@@ -12,17 +12,17 @@ <h3>NavCoin</h3>
                     </div>
                     <p>NavCoin released under the <a href="https://opensource.org/licenses/mit-license.php">MIT license</a></p>
                     <div class="themeix-social-area">
-                        <a class="themeix-tool" href="">
+                        <a class="themeix-tool" href="https://twitter.com/NavCoin">
```

Comment:
i think these footer changes should dissapear when your other branch is merged to master.